lib/storage/s3.js

```diff
@@ -105,8 +116,16 @@ module.exports = function() {
       var params = {
         Key: cleanKey(path)
       };
-      var inputStream = client.getObject(params).createReadStream();
-      inputStream.pipe(outputStream);
+      var request = client.getObject(params);
+      var inputStream = request.createReadStream(inputStream);
```

Comment:
You create `inputStream` from `inputStream`. Is this intended? Or it might be from another path?

Reply:
Interesting. Good question.

Reply:
Good catch. This method actually takes no arguments, which is why it didn't crash. I will remove the argument. Thanks.